cfgov/jinja2/v1/_includes/organisms/expandable.html

```diff
@@ -33,62 +33,46 @@
 
 {% macro expandable(value, expandable_cue_additional_text='') %}
 
-<div data-qa-hook="expandable" class="o-expandable
-            o-expandable__expanded
-            {{ 'o-expandable__borders' if value.is_bordered else '' }}
-            {{ 'o-expandable__midtone' if value.is_midtone else '' }}"
-            {{ 'data-state=expanded' if value.is_expanded else '' }}>
-
-    <button type="button" class="o-expandable_target">
-        <div class="o-expandable_header">
-            <span class="o-expandable_header-left
-                         o-expandable_label">
-                {{ value.label }}
+<div data-qa-hook="expandable"
+     class="o-expandable
+            o-expandable__padded
+            {{ 'o-expandable__background' if value.is_midtone else '' }}
+            {{ 'o-expandable__border' if value.is_bordered else '' }}">
+
+    <button class="o-expandable_header o-expandable_target">
+        <span class="h4 o-expandable_header-left o-expandable_label">
+            {{ value.label }}
+        </span>
+        <span class="o-expandable_header-right o-expandable_link">
+            <span class="o-expandable_cue o-expandable_cue-open">
+                {{ 'Show' if true else 'Mostrar' }}
```

Comment:
What's going on here? Isn't this conditional pointless?

Reply:
Sorry, I need to replace the code that was here, I was hoping we could change the language on the backend, but I'm not so sure that's the right solution now.